packages/dart/lib/src/hub.dart

```diff
@@ -573,6 +576,25 @@ class Hub {
     return NoOpSentrySpan();
   }
 
+  Span startSpan(
+    String name, {
+    SpanParent parentSpan = const SpanParent.active(),
+    bool active = true,
```

Comment:
Also having both `SpanParent.active()` and `active` parameter might be confusing. WDYT?